packages/url/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/url",
-	"version": "2.0.2",
+	"version": "2.1.0",
```

Comment:
We should never update version manually, the same applies to the flow without Lerna, npm takes care of it with [npm version](https://docs.npmjs.com/cli/version) command.